featuretools/variable_types/variable.py

```diff
@@ -1,7 +1,12 @@
 import numpy as np
 import pandas as pd
 
-from featuretools.utils.gen_utils import find_descendents
+from featuretools.utils.gen_utils import camel_to_snake, find_descendents
+
+
+class ClassNameDescriptor(object):
+    def __get__(self, instance, class_):
```

Comment:
a brief docstring  mentioning this is a descriptor and summarizing the purpose of the class could be useful here